setup.py

```diff
@@ -36,21 +37,27 @@ def read(*rnames):
           + '\n\n' +
           read('CHANGES.txt')
           ),
-      keywords = "zope3 skin zmi",
-      classifiers = [
+      keywords="zope3 skin zmi",
+      classifiers=[
           'Development Status :: 5 - Production/Stable',
           'Environment :: Web Environment',
           'Intended Audience :: Developers',
           'License :: OSI Approved :: Zope Public License',
           'Programming Language :: Python',
+          'Programming Language :: Python:: 2.7',
+          'Programming Language :: Python:: 3.5',
```

Comment:
Well, we probably could.  But I'm not using Python 3.4 and have no plans to---PyPy won't have a Python 3.4 release, just a 3.5 release, and for me that's enough reason to skip 3.4 (and 3.3) entirely. Since this is the first time Python 3 has been addressed here, it didn't seem like people were clamoring 3.4 support. So it just seemed simpler to not put it in---plus it saves time on Travis, in tox, etc.

But if you feel strongly, I can add it to all those places.